CMakeLists.txt

```diff
@@ -772,7 +772,7 @@ endif(SUPPORT_X11 AND NOT ALLEGRO_RASPBERRYPI)
 if(WIN32)
     list(APPEND LIBRARY_SOURCES ${ALLEGRO_SRC_WIN_FILES})
     list(APPEND PLATFORM_LIBS
-        kernel32 user32 gdi32 comdlg32 ole32 winmm psapi shlwapi
+        kernel32 user32 gdi32 comdlg32 ole32 winmm psapi shlwapi Shcore
```

Comment:
I could actually use the below trick to set this and include the header dynamically without breaking older windows support:

https://github.com/neuro-sys/allegro5/blob/ad8bb20aba538bcb29cfcb2f4cb56c1d61552004/CMakeLists.txt#L268-L275